HLTrigger/Configuration/python/Eras.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+# import the relevant eras from Configuration.Eras.*
+from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
+
+def modifyHLTforEras(fragment):
+    """load all Eras-based customisations for the HLT configuration"""
+
+    # modify the HLT configuration for the Phase I pixel geometry
+    from HLTrigger.Configuration.customizeHLTTrackingForPhaseI2017 import customizeHLTPhaseIPixelGeom
+    trackingPhase1.toModify(fragment, customizeHLTPhaseIPixelGeom)
+
+    # modify the HLT configuration to run the Phase I tracking in the particle flow sequence
+    from HLTrigger.Configuration.customizeHLTTrackingForPhaseI2017 import customizeHLTForPFTrackingPhaseI2017
+    trackingPhase1.toModify(fragment, customizeHLTForPFTrackingPhaseI2017)
```

Comment:
I'd suggest to use `phase1Pixel` sub-era instead of `trackingPhase1`. The former is in all `Run2_2017`, `Run2_2017_trackingRun2`, and `Run2_2017_trackingPhase1PU70` eras, while the latter is only in the first one.

More philosophically, `phase1Pixel` specifies the 2017 pixel detector, while `tracking*` are used to specify the offline tracking configuration to be run (which is, at least technically, not that much tied to a specific detector, at least between phase0 and phase1).